### ISSUE TITLE
Redirect feature in Landing Page

### ISSUE THREAD

**Issue by tr0tter** [gophish/gophish#682]
Hi,

I managed to run this tool successfully, however the redirect feature does not seem to work for me. I entered a webpage for e.g. www.google.com as my redirect page after the user has submitted their username. (Please note: I only capture username and not the password).

I notice that after I enter my login details in the phishing page the structure of the redirect is as follows:

www.phishingpage.com/www.google.com

Based on that it tries to identify www.google.com as a subset/page of the www.phishingpage.com domain and returns a 403 response. I even checked this in the terminal by looking at the raw HTTP requests and responses.

From what I have read a 302 is supposed to be issued to redirect to the page you had specified in the options while configuring the Landing Page. 

Am I doing something wrong, or have I missed to include any parameters while creating the landing page?

Thanks!

Tr0tter

**Comment by jordan-wright** [comment]
Does that happen if you make the redirect URL https://www.google.com?

**Comment by jordan-wright** [comment]
Trying it on my own Gophish instance, I've confirmed that putting the URL scheme (e.g. https://) in front of the URL does appear to fix this issue.

I've updated our docs to include a special note that encourages users to include the URL scheme. Since our placeholder currently includes a full URL, I'm considering this issue closed.

@tr0tter let us know if you continue to run into any issues and we will be happy to troubleshoot.
